CHANGES.md

```diff
@@ -1,4 +1,4 @@
-# 3.0.0 UNRELEASED
+# 3.0.0 (2018-11-19)
 - [FIXED] Expose `BasePlugin` type in Cloudant client.
 - [REMOVED] Remove nodejs-cloudant TypeScript type definitions for
   `db.search`. These definitions are now imported directly from Nano.
```

Comment:
I think we need a reference to the migration guide here. Similar to what we did for [version 2](https://github.com/cloudant/nodejs-cloudant/blob/master/CHANGES.md#201-2018-02-14). I think it could be associated with the `[UPGRADED]` nano point, but perhaps we should add a `[BREAKING CHANGE]` that talks about the return type change and points to the migration guide from there e.g.

```markdown
- [BREAKING CHANGE] Due to the Nano 7.x upgrade return types are now `Promise`s. _See [api-migration.md](https://github.com/cloudant/nodejs-cloudant/blob/master/api-migration.md#2x--3x)
  for migration details._
```
wdyt?